revolv/payments/models.py

```diff
@@ -19,6 +21,13 @@ def repayments(self, admin=None, project=None, queryset=None):
             queryset = queryset.filter(admin=admin).order_by('created_at')
         if project:
             queryset = queryset.filter(project=project).order_by('created_at')
+        if start_date and not end_date:
```

Comment:
Not sure why these two if statements need to be here. Can't you just filter on end date if you see an end date and filter on start date if you see a start date?